.github/workflows/ci.yml

```diff
@@ -1678,6 +1678,22 @@ ${{ hashFiles('.pre-commit-config.yaml') }}"
         run: breeze free-space
       - name: "Start ARM instance"
         run: ./scripts/ci/images/ci_start_arm_instance_and_connect_to_docker.sh
+      - name: "Retrieve DEFAULTS from the _initialization.sh"
```

Comment:
We don't need it here. This step is actually a bit different than the other steps. It is never run in PR or direct pushes to a branch. It's purpose is to refresh cache. 

It is only run in "main" and "vX-Y-test" branch when directly pushing those branches (so either when we merge to main or when we cherry-pick to vX-Y-test".

This is done by: `if: needs.build-info.outputs.mergeRun == 'true'`

It starts ARM instance in the background and runs multi-platform build. 

The main reason why it failed here is because the ARM build failed due to JPype not releasing the "sdist" package.

Generally speaking - we do not need this "retrieving defaults" because the "retrieve defaults" is only needed in PR builds from forks for security reason - when we have workflow_run using "main" code, we want to find out which are the values coming from the PR - and this code does it. This code will never be executing in PR from fork so we do not need it here.